settings.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Plugin configuration page
+ * Based on: https://vedovini.net/2015/10/using-the-wordpress-settings-api-with-network-admin-pages/
+ */
+
+add_action( 'init', array( 'Multisite_Enhancements_Settings', 'init' ) );
+
+class Multisite_Enhancements_Settings {
+
+	/**
+	 * Default options settings
+	 */
+	static protected $default_options = array(
+		'remove-logo'         => '1',
```

Comment:
I would change them to boolean values, `1`, not a string `'1'`.

Reply:
Ok, done in https://github.com/bueltge/wordpress-multisite-enhancements/pull/63/commits/62b2e77f7c438c952204639cc4193ab4e64914d0